server/src/main/java/com/example/controller/PerkManagerController.java

```diff
@@ -30,6 +30,15 @@ public ResponseEntity<Perk> addNewPerk(@RequestBody Perk perk) {
         return ResponseEntity.ok(savedPerk);
     }
 
+    @GetMapping("/all")
```

Comment:
I dont think it is necessary to add this, /api/perks is set up to get all perks, now with this it does the same thing but at /api/perks/all